test/spec/dataviews/histogram-dataview-model.spec.js

```diff
@@ -18,6 +20,47 @@ describe('dataviews/histogram-dataview-model', function () {
     });
   });
 
+  it('should not listen any url change from the beginning', function () {
+    expect(this.model.listenTo.calls.count()).toBe(5);
```

Comment:
`expect(this.model.listenTo.calls.count()).toBe(5);` is not very descriptive IMO and it makes this test case not very laser focused :)

What about?
```
var callback = jasmine.createSpy('callback');
this.model.bind('change:url', callback);
this.model.set('url', 'http://example.com');
expect(callback).not.toHaveBeenCalled();
```